src/components/ResponsiveList/__stories__/responsiveList.stories.js

```diff
@@ -39,7 +39,13 @@ const DefaultExampleTemplate = (responseListProps) => {
     <SplitButton size={SplitButton.sizes.MAIN} marginRight secondaryDialogContent={<SecondaryContentComponent />}>
       Add Item
     </SplitButton>
-    <div className="responsive-lst-search">
+    <div className="responsive-lst-search" responsiveListPlaceholder={<Search
+      id={"search-icon-inside"}
+      size={Search.sizes.MEDIUM}
+      placeholder="search for content"
+      iconName={() => <SearchIcon />}
+      secondaryIconName={() => <CloseSmall />}
+    />}>
       <Search
```

Comment:
The invisible now has this id:
![image](https://user-images.githubusercontent.com/2057934/126898377-11f5365d-9734-4916-869a-503c55f370a7.png)
so no duplicates ids